### ISSUE TITLE
EyeDropper getting the wrong color

### ISSUE THREAD

**Issue by gundamftw** [jodhman/react-eyedrop#5]
The plugin in giving me the wrong color, I have body background color set to lightgray and the eyedropper is reading it as rgb(255, 255, 255).

````javascript
class App extends Component {
	handleChangeColor ({ rgb, hex }) {
		console.log(rgb);
	}
	render() {
		return (
			<div>				
				<EyeDropper
					onChange={this.handleChangeColor} />
			</div>
		);
	}
}

````

**Comment by gundamftw** [comment]
And the radius result is wrong too, I sampled on a "white" part of an image and it is reading it as rgb(164, 159, 140).

````javascript
<EyeDropper pickRadius={{ unit: 'radius', amount: 1 }}	/>
````
Setting amount to 0 gives a blow up number.

**Comment by jodhman** [comment]
Uh-oh. Smells like a bug in the dependencies. Will have to change the way pixels are translated into colors.

Setting amount to 0 with unit radius should be handled with an error. That's a bug, right there.

**Comment by jodhman** [comment]
Setting amount to 0 now correctly throws an error.
The dependency bug will be handled as soon as a better replacement is found.

**Comment by jodhman** [comment]
Saw that the canvas-to-color dependency had a major update which contained a bugfix.
Updated the dependency and released version 3.3.2 containing the update.
Let's see if this resolved the issue.

**Comment by gundamftw** [comment]
Nope, all the color are still wrong :(
